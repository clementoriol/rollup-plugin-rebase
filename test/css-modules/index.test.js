import { bundle, clean, list, read } from "../util"

const root = __dirname

test("CSS Modules", async () => {
  await bundle(root, "index.js", "output/index.js")

  expect(await list(root, "output")).toMatchSnapshot()
  expect(await read(root, "output/index.js")).toMatchSnapshot()
  expect(await read(root, "output/KKaZlCcl.module.css")).toMatchSnapshot()

  await clean(root, "output")
})
