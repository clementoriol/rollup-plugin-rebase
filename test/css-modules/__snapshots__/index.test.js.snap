// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CSS Modules 1`] = `
"KKaZlCcl.module.css
KKaZlCcl.module.css.map
bsdudxaF.md
index.js"
`;

exports[`CSS Modules 2`] = `
"import content from './bsdudxaF.md';
import css from './KKaZlCcl.module.css';

function index() {
  return content + css
}

export default index;
"
`;

exports[`CSS Modules 3`] = `
"body {
  font-size: 20px;
}

/*# sourceMappingURL=KKaZlCcl.module.css.map */"
`;
